sccp/imports/scRNA.py

```diff
@@ -135,20 +163,6 @@ def load_lupus_data(third_axis="ind_cov", every_n=4):
     # keep only n observations
     X = X[::every_n, :]
 
-    # get cell types -> only stay true if order of cells doesn't change after this
-    cell_types = X.obs["cg_cov"].reset_index(drop=True)
-
-    # get color mapping for patients by SLE status
-
-    status = (
-        X.obs[["ind_cov", "SLE_status"]]
-        .sort_values(by="ind_cov")
-        .drop_duplicates("ind_cov")
-    )
-
-    lut = {"SLE": "c", "Healthy": "m"}
-    row_colors = status["SLE_status"].map(lut)
-
     assert np.all(np.isfinite(X.X.data))  # this should be true
 
-    return tensorFy(X, third_axis), cell_types, row_colors
+    return tensorFy(X, 'sample_ID'), X.obs
```

Comment:
I could also just output the entire anndata object `X`, but this (`X.obs`) might take up a bit less room and (at the moment) is the only part I'm actually using

Reply:
I don't believe there would be a reason to need the X unless you wanted to look at the raw data